### ISSUE TITLE
GTK2 build fails on subsequent compilation(s)

### ISSUE THREAD

**Issue by savchenko** [neeeeow/Bluecurve#12]
```
Compiling GTK 2 engine...
mkdir: cannot create directory ‘build’: File exists
CMake Warning:
  Ignoring extra path from command line:

   ".."

CMake Error: The source directory "/home/user/code/Bluecurve/engine" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
make: *** No targets specified and no makefile found.  Stop.
```

Perhaps a cleanup step in the `install.sh` would be a good addition.
Something along the lines of `git reset --hard && git clean -diX`.

**Comment by neeeeow** [comment]
The issue here is that the script doesn't delete any old build files before trying to compile the GTK 2 engine. I'll fix this issue soon when I have some free time.

**Comment by neeeeow** [comment]
This issue has been fixed in commit [5af6f5b](https://github.com/neeeeow/Bluecurve/commit/5af6f5b55bec162bab0b15007054dee0d51b4831).

Thanks for reporting this!
